pkg/reporter/markdown/markdown_test.go

```diff
@@ -63,3 +63,26 @@ func TestMarkdownBuilder(t *testing.T) {
 		})
 	}
 }
+
+func TestMarkdownBuildPlainText(t *testing.T) {
```

Comment:
Is `AAAA # BBB` valid markdown? Hope the regex won't end up replacing `#` which in this case is intended as a character within the text.

Reply:
yes, it won't replace if its in between, i'll add this in testcase